src/targets/common.ini

```diff
@@ -17,11 +17,11 @@ build_flags_tx = -DTARGET_TX=1 ${common_env_data.build_flags}
 build_flags_rx = -DTARGET_RX=1 ${common_env_data.build_flags}
 
 [radio_900]
-build_flags = -DRADIO_900=1
+build_flags = -DRADIO_SX127X
```

Comment:
Why this could not still be `-DRADIO_SX127X=1`?
That way usage could be simplified to `#if RADIO_SX127X` ...